remotes/locationIndependentIdentifier.json

```diff
@@ -0,0 +1,11 @@
+{
+    "refToInteger": {
+        "$ref": "#foo"
+    },
+    "definitions": {
+        "A": {
+            "id": "#foo",
```

Comment:
this should probably be $id, though it's a little ambiguous which draft the remotes are supposed to be

Reply:
Good catch! I copied the schema from Draft 4 and didn't notice that it should be named differently. Would it be better to have two schemas - one for Draft 4 with `id` and another for later drafts with `$id`?

Reply:
I think this came up in a previous PR as well and we talked about making version-specific remotes directories, to avoid issues like this in the future. That's a potentially-breaking change for consumers though.

Reply:
For now I did the [hacky thing](https://github.com/json-schema-org/JSON-Schema-Test-Suite/commit/cf78d97d069f351a652460bafa1531aa64363b85). Indeed we should unfortunately solve this in general at some point...

Reply:
My test runner broke as a result of this change. Can we please talk about a non-hacky solution?

Reply:
How could it break from such a thing? What I meant by hacky was simply hacky for us as maintainers of the suite. But yes sure happy to brainstorm that too.